src/System.Data.Common/src/Configurations.props

```diff
@@ -5,6 +5,7 @@
       uap-Windows_NT;
       netcoreapp;
       netfx;
+      uapaot-Windows_NT;
```

Comment:
General question here, do we really want uap to build against contracts but uapaot against System.Private.Xml? Why not just make the uap configuration to build against System.Private.Xml?